.github/workflows/BibtoolCI.yml

```
@@ -25,7 +25,12 @@ jobs:
           sudo apt-get install -y bibtool
       - name: Execute bibtool and check for changes
         run: |
-          bibtool docs/oscar_references.bib -o docs/oscar_references.bib
+          git config user.name "Bibtool CI Job"
+          git config user.email "bibtool.ci.job@noreply.github.com"
+          touch bibtool.log
+          git add bibtool.log
+          git commit -m "Add bibtool.log"
+          bibtool docs/oscar_references.bib -o docs/oscar_references.bib 2>&1 | tee bibtool.log
```

If all you want to do is to check whether the log file is empty, you can also do it like this:
```suggestion
          bibtool docs/oscar_references.bib -o docs/oscar_references.bib 2>&1 | tee bibtool.log
          test -s bibtool.log
```

---

This is missing a negation, but I now used something very similar